docs/NooBaaNonContainerized/Health.md

```diff
@@ -0,0 +1,350 @@
+          "storage_path": "/tmp/nsfs_root/bucket2"
+        }
+      ],
+      "error_type": "PERSISTENT"
+    }
+  }
+}
+```
+
+### Example Output Details - 
+
+- NooBaa endpoint response:
+  - Curl command to NooBaa endpoint returned successfully, therefore, NooBaa health reports RUNNING.
+
+- invalid_accounts: 
+  - The new_buckets_path of account_invalid's doesn't exist in the file system, therefore, NooBaa health reports STORAGE_NOT_EXIST.
+  - The uid/gid/user of account_inaccessible has insufficient read and write permissions to its new_buckets_path, therefore, NooBaa health reports ACCESS_DENIED.
+
+- invalid_buckets:
+  - The config file of bucket1 is invalid. Therefore, NooBaa health reports INVALID_CONFIG.
+  - The underlying file system directory of bucket3 is missing. Therefore, NooBaa health reports STORAGE_NOT_EXIST.
+
+
+
+
+## Health Errors
+
+If the Health CLI encounters issues, the following errors will appear in the health output. If any of these error codes are present in the health status, the overall status will be flagged as `NOTOK`.
+
+### General Errors
+
+#### 1. Invalid NooBaa Service
+  - Error code: `NOOBAA_SERVICE_FAILED`
+  - Error message: NooBaa service is not started properly, Please verify the service with status command.
+  - Reasons:
+    - NooBaa service is not started properly.
+    - Stopped NooBaa service is not removed.
+  - Resolutions:
+    - Verify the NooBaa service is running by checking the status and logs command.
+      ```
+      systemctl status noobaa.service
+      journalctl -xeu noobaa.service
+      ```
+      If the NooBaa service is not started, start the service
+      ```
+      systemctl enable noobaa.service
+      systemctl start noobaa.service
+      ```
+
+#### 2. Missing Forks
+  - Error code: `NOOBAA_ENDPOINT_FORK_MISSING`
+  - Error message: One or more endpoint fork is not started properly. Verify the total and missing fork count in response.
+  - Reasons:
+    - One or more endpoint fork is not started properly.
+    - Number of workers running is less than the configured `forks` value.
+  - Resolutions:
+    - Restart the NooBaa service and also verify NooBaa fork/s is exited with an error within the logs.
+      ```
+      systemctl status rsyslog.service
+      journalctl -xeu rsyslog.service
+      ```
+
+#### 3. NooBaa Endpoint Is Not Running
+  - Error code: `NOOBAA_ENDPOINT_FAILED`
+  - Error message: S3 endpoint process is not running. Restart the endpoint process.
+  - Reasons:
+    - NooBaa endpoint process is not running, and it's not able to respond to any requests.
+  - Resolutions:
+    - Restart the NooBaa service and verify NooBaa process is exited with errors within the logs.
+      ```
+      systemctl status rsyslog
+      journalctl -xeu rsyslog.service
+      ```
+
+#### 5. Unknown Error
+  - Error code: `UNKNOWN_ERROR`
+  - Error message: An unknown error occurred.
+  - Reasons:
+    - Unknown.
+  - Resolutions:
+    - Unknown.
```

Comment:
We better advise on what to do in such a case or refer to the troubleshooting part.